jsonxx.cc

```diff
@@ -610,7 +610,7 @@ namespace json {
         return input;
     }
 
-    std::string tag( unsigned format, unsigned depth, const std::string &name, const jsonxx::Value &t, PrintMode printMode) {
+    std::string tag( unsigned format, unsigned depth, const std::string &name, const jsonxx::Value &t, PrintMode printMode, int floatPrecision = std::numeric_limits<long double>::digits10 + 1) {
```

Comment:
Surprisingly, the function signature is `std::setprecision( int n );`, so I'm using `int` here.